### ISSUE TITLE
can't activate sinatra

### ISSUE THREAD

**Issue by jdownie** [auxesis/visage#71]
I've followed the instructions on the web site to install visage-app, and I get the following error...

```
root@china:/var/lib/gems/1.8/bin# ./visage-app start
/usr/lib/ruby/1.8/rubygems.rb:233:in `activate': can't activate sinatra (~> 1.1.3, runtime) for ["visage-app-0.9.6"], already activated sinatra-1.2.6 for ["visage-app-0.9.6"] (Gem::LoadError)
     from /usr/lib/ruby/1.8/rubygems.rb:249:in `activate'
     from /usr/lib/ruby/1.8/rubygems.rb:248:in `each'
     from /usr/lib/ruby/1.8/rubygems.rb:248:in `activate'
     from /usr/lib/ruby/1.8/rubygems.rb:1082:in `gem'
     from ./visage-app:18
root@china:/var/lib/gems/1.8/bin# 
```

This is on Ubuntu Server 11.04. I admit that i'm completely new to ruby, so this might be a very simple ruby question, but something about the instructions on the web site doesn't seem to agree with Ubuntu server - for me at least.

**Comment by greenmang0** [comment]
Well I solved the issue. I found there were ruby gems install in "/var/lib/gems/1.8/" as well as in "~/.gem/" directory.
I first tried installing "visage-app" as a non-sudoer. Which first downloaded all the dependencies in ~/.gem directory and then gave a warning. 

WARNING:  Installing to ~/.gem since /var/lib/gems/1.8 and
          /var/lib/gems/1.8/bin aren't both writable.
WARNING:  You don't have /home/odin/.gem/ruby/1.8/bin in your PATH,
          gem executables will not run.
Building native extensions.  This could take a while...
ERROR:  Error installing visage-app:
        visage-app requires sinatra (~> 1.1.3, runtime)

So instead of adding that "bin/" directory to my path, I used sudo to install "visage-app", which again installed all the dependencies in "/var/lib/gems/1.8/". After the successful installation when I tried to run "visage-app start" then "can't activate sinatra..." error was shown. 

Actually there's no need to install "visage-app" as a superuser at all. Install it without "sudo". It will install all the dependencies/ gems in "~/.gem" directory. Use "gem list" to list the gems installed on your system. "visage-app 0.9.6" has following dependencies.

errand (0.7.3)
haml (3.0.13)
rack (1.3.0)
sinatra (1.1.3)
tilt (1.2.2)
visage-app (0.9.6)
yajl-ruby (0.8.2)

Other version of these dependencies cause problem. So remove unwanted gems using "gem uninstall gem_name". And you done. Now "visage-app" can run without any errors. :)
